src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Host, h, Prop, Watch, State } from "@stencil/core";
+
+@Component({
+  tag: "radio-group",
+  styleUrl: "radio-group.css",
+  shadow: true,
+})
+export class RadioGroup {
+  @Prop({ reflect: true }) list: Array<string>;
```

Comment:
@anveshmekala have you explored slotting the list items instead of a property? Could you explain your choice of using a property here vs slotting a list of items?